post_processing/plot/plot_everything.jl

```diff
@@ -0,0 +1,236 @@
+# plot_everything.jl: a simple Julia script to make a suite of plots for quick inspection as
+# part of CI.
+
+if length(ARGS) < 1
+    error("Run as plot_everything.jl <path of datadir>")
+end
+
+using Statistics # Used for `mean`
+using NCDatasets
+using CairoMakie
```

Comment:
Can we use the `import` pattern here instead, and qualify where things are coming from?